integreat_cms/cms/views/linkcheck/linkcheck_actions.py

```diff
@@ -0,0 +1,43 @@
+"""
+This module contains view actions related to link check.
+"""
+import json
+
+from django.http import JsonResponse
+
+from ...utils.linkcheck_utils import replace_links
+
+
+# pylint: disable=unused-argument
+def replace_region_links(request, region_slug):
+    """
+    Replace region links
+
+    :param request: The current request
+    :type request: ~django.http.HttpRequest
+
+    :param region_slug: slug of a region
+    :type region_slug: str
+
+    :return: JSON response which indicates error or success
+    :rtype: ~django.http.JsonResponse
+    """
+    region = request.region
+    data = json.loads(request.body.decode("utf-8"))
+    search = data.get("oldSlug")
+    replace = data.get("newSlug")
+
+    if not search or not replace:
+        return JsonResponse(
+            data={
+                "response": "Fail",
+            },
+            status=400,
+        )
+
+    replace_links(search, replace, region, commit=True, only_internal=True)
```

Comment:
It would be really amazing if we could have something like a confirmation-stage, where the replacement is run with `commit=False` and the changes are only printed out, and not yet performed... But I guess this would require some refactoring of the existing function, so let's open a new issue for this once this is done :smile: